__tests__/meta.test.ts

```diff
@@ -44,7 +44,7 @@ describe('isRawStatement', () => {
     ['{{ raw }}', true],
     ['{{ raw}}', true],
     ['{{raw}}', true],
-  ])('given %p pattern', async (pattern: string, expected: boolean) => {
+  ])('given %s pattern', async (pattern: string, expected: boolean) => {
```

Comment:
`%p` is a jest positional formatting placeholder that [uses pretty-format to stringify the parameter](https://jestjs.io/docs/api#1-testeachtablename-fn-timeout). IOW, unless there's a compelling reason to change it to `%s`, I don't see why that needs to be messed with.